examples/notebooks/create_deployment.ipynb

```diff
@@ -12,48 +23,288 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load config from default location."
+   ]
+  },
+  {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "client.Configuration().host=\"http://localhost:8080\""
+    "config.load_kube_config()\n",
+    "extensions = client.ExtensionsV1beta1Api()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Deployment object"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "extensions = client.ExtensionsV1beta1Api()"
+    "deployment = client.models.extensions_v1beta1_deployment.ExtensionsV1beta1Deployment()"
```

Comment:
@djkonro actually you should change this to `deployment = client.ExtensionsV1beta1Deployment()`